src/components/BookList.js

```diff
@@ -1,17 +1,18 @@
 /* eslint-disable max-len */
-import { React, useState } from 'react';
+import React from 'react';
+import { useSelector } from 'react-redux/es/exports';
 import Books from './Books';
 import Form from './Form';
 
 function BookList() {
-  const [books] = useState([{ id: '1', title: 'Animal Farm', author: 'George Orwell' }]);
+  const books = useSelector((state) => state.books);
```

Comment:
List of books displays books from Redux store (by using `useSelector` function) ✔